src/app/settings/settings.component.ts

```diff
@@ -46,4 +47,34 @@ export class SettingsComponent implements OnInit {
     debug('setting new server to ' + value);
     this.data.changeFhirMicroService(value);
   }
+
+  async createQuestionnaireResponse() {
```

Comment:
how about `setQuestionnaireDefaultResponse`?

Reply:
Yes, this is a better naming or maybe I even include RadOrder in the name as this is only for this specific questionnaire.